### PR TITLE
Make `single_stream_expected_latency_ns` configurable in benchmark_setting for Android app

### PR DESCRIPTION
**Pull request by anhappdev**
Closes https://github.com/mlcommons/mobile_app_open/issues/291

### CODE REVIEW

**Comment by anhappdev** [comment]
After updating `benchmark_setting` you may need to uninstall then reinstall the app for the new settings to take effect. I did not find out why.

File: android/cpp/proto/backend_setting.proto [hunk 1]
```diff
@@ -47,6 +47,8 @@ message BenchmarkSetting {
   required string configuration = 6;
   // batch_size is passed to the backend
   optional int32 batch_size = 7;
+  // single_stream_expected_latency_ns is passed to the MlperfDriver, default to 1M, max 1M
+  optional int32 single_stream_expected_latency_ns = 8 [default = 1000000];
```
Comment (anhappdev) [inline android/cpp/proto/backend_setting.proto:51]: default value for `single_stream_expected_latency_ns` is 1M

**Review by d-uzlov — APPROVED** [review]

**Review by freedomtan — DISMISSED** [review]
tested with #345. It works as expected on MTK internal devices

File: android/docker/mlperf_mobile/Dockerfile [hunk 2]
```diff
@@ -67,7 +67,7 @@ RUN yes | sdkmanager \
     "extras;android;m2repository" \
     "extras;google;m2repository" \
     "extras;google;google_play_services" \
-    "build-tools;29.0.3"
+    "build-tools;30.0.3"
```
Comment (mohitmundhragithub) [inline android/docker/mlperf_mobile/Dockerfile:70]: The build tool on line 58 is 29.0.3. Is it ok to change it to 30.0.3 here without changing in line 58 as well?

**Review by freedomtan — APPROVED** [review]
@anhappdev please merge this related modifications.

**Pull request merged**